wagtailcache/cache.py

```diff
@@ -184,6 +195,33 @@ def clear_cache() -> None:
     if wagtailcache_settings.WAGTAIL_CACHE:
         caches[wagtailcache_settings.WAGTAIL_CACHE_BACKEND].clear()
 
+def clean_uri(uri):
+    """
+    delete Trailingslash if available
+    """
+    if uri[-1] == '/':
+        uri = uri[:-1]
+
+    return uri
+
+def purge_cache(urls) -> None:
```

Comment:
Could you merge this functionality into the existing `clear_cache()` function? It can take an optional list of URLs, default to None, for backwards compatibility. E.g:

```python
def clear_cache(urls: List[str] = None) -> None:
```